packages/text-sets/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "declarationDir": "dist-types"
+  },
+  "include": ["src/**/*"]
```

Doesn't this config needs `references` to both the react and types packages?

```suggestion
  "references": [{ "path": "../react" }, { "path": "../types" }],
    "include": ["src/**/*"]
```

I thought it was required to make the types resolve correctly between packages? I do see that the tsc bundler runs just fine without it, so I'm wondering why we have to do it elsewhere actually? @swissspidy, any idea?

---

I'd have to look into it, but see https://www.typescriptlang.org/docs/handbook/project-references.html#what-is-a-project-reference for what this is supposed to do.

---

I added in the references. It doesnt seem to hurt anything. If we should remove it then we should remove it other places.